components/DraggableCourse.tsx

```diff
@@ -65,13 +65,8 @@ export const Course: FC<DragCourseType> = memo(function Course({
       data-testid="course"
       className={clsx(
         "CourseText",
-        warningYellowColor !== undefined &&
-          warningRedColor === undefined &&
-          warningOrangeColor === undefined &&
-          "CourseWarningYellow",
-        warningOrangeColor !== undefined &&
-          warningRedColor === undefined &&
-          "CourseWarningOrange",
+        warningYellowColor !== undefined && warningRedColor === undefined && warningOrangeColor === undefined && "CourseWarningYellow",
```

Comment:
Does `&& "CourseWarningYellow"` actually do anything? I believe that it shouldn't do anything since `"CourseWarningYellow"` always evaluates to true.

Reply:
This is a very strange way of creating a class name for the element that changes what css class is used for the color.

Reply:
I hate it but it's a carryover from last semester and would require quite a bit of change to fix